admin-site/src/App.svelte

```diff
@@ -4,22 +4,24 @@
 -->
 
 <script>
-  // import Login from "./Login.svelte";
-  // import Slider from "./Slider.svelte";
   import moment from "moment";
   import { onMount } from 'svelte';
   import { createEventDispatcher } from "svelte";
-
-
+  import { writable } from 'svelte/store';
+  import Page from './ParentPage.svelte';
+  import Slider from './Slider.svelte';
+  import Button from './Button.svelte';
+  // import {goBackAPage} from './ButtonActions.svelte'
+  import Modal from './Modal.svelte';
+  import OccasionsList from './OccasionsList.svelte';
   
-
-  let events = []
+  let storedEvents;
   let gotEvents = false;
+  //holds a store of events
+  $: events = [];
 
   // // TODO this needs to pickup an environment somehow (dev/staging/prod)
-  // // let serverURL = "https://staging.cohort.rocks/api/v2"
-  
-  // let serverURL = "http://localhost:3000/api/v2";
+  //serverURL populated with dropdown at login - default is staging.cohort.rocks
```

Comment:
Can you check the window.location for 'localhost' or '.local'? If found, you could set the default serverURL to that local URL (i.e. 'http://localhost:3000' or 'http://some-computer-name.local')? That way we can still override using the dropdown but local dev is a bit easier.

Reply:
ok yeah. I have something implemented now.